code/modules/urist/items/uristweapons.dm

```diff
@@ -115,4 +115,95 @@ Please keep it tidy, by which I mean put comments describing the item before the
 	edge = 0
 	lock_picking_level = 2
 
-///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
\ No newline at end of file
+///obj/item/weapon/gun/launcher/crossbow/bow //gonna mess with this
+
+
+//RS Weapons
+
+
+
+/obj/item/weapon/BronzeDagger
```

Comment:
these should be consistent with the other weapons, add them all to /obj/item/weapon/urist.